dev-proxy/CommandHandlers/CertRemoveCommandHandler.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace DevProxy.CommandHandlers;
+
+public static class CertRemoveCommandHandler
+{
+    public static void RemoveCert(ILogger logger)
+    {
+        logger.LogTrace("RemoveCert() called");
+
+        try
+        {
+            logger.LogInformation("Uninstalling the root certificate...");
+            ProxyEngine.ProxyServer.CertificateManager.RemoveTrustedRootCertificate(machineTrusted: false);
```

Comment:
I noticed, that even if I say 'No' in the prompt confirming if I want to remove the certificate, it's still getting removed. Let's double check that the code is working as expected.

Reply:
That dialogue is shown by Win API, it appears to confirm removal from Trusted Root CA. A certificate from Personal does not require confirmation.